docs/src/user-guide/command-line-interface.md

```diff
@@ -241,71 +229,88 @@ npx eslint --rule 'jquery/dollar-sign: error'
+This option instructs ESLint to try to fix as many issues as possible. The fixes are made to the actual files themselves and only the remaining unfixed issues are output.
+
+* **Argument Type**: No argument.
+
+Not all problems are fixable using this option, and the option does not work in these situations:
 
 1. This option throws an error when code is piped to ESLint.
 1. This option has no effect on code that uses a processor, unless the processor opts into allowing autofixes.
 
 If you want to fix code from `stdin` or otherwise want to get the fixes without actually writing them to the file, use the [`--fix-dry-run`](#--fix-dry-run) option.
 
+##### `--fix` example
+
+```shell
+npx eslint --fix file.js
+```
+
 #### `--fix-dry-run`
 
-This option has the same effect as `--fix` with one difference: the fixes are not saved to the file system. This makes it possible to fix code from `stdin` (when used with the `--stdin` flag).
+This option has the same effect as `--fix` with the difference that the fixes are not saved to the file system. Because the default formatter does not output the fixed code, you'll have to use another formatter (e.g. `--format json`) to get the fixes.
 
-Because the default formatter does not output the fixed code, you'll have to use another one (e.g. `json`) to get the fixes. Here's an example of this pattern:
+* **Argument Type**: No argument.
 
-```shell
-getSomeText | npx eslint --stdin --fix-dry-run --format=json
-```
+This makes it possible to fix code from `stdin` when used with the `--stdin` flag.
 
 This flag can be useful for integrations (e.g. editor plugins) which need to autofix text from the command line without saving it to the filesystem.
 
+##### `--fix-dry-run` example
+
+```shell
+getSomeText | npx eslint --stdin --fix-dry-run --format json
+```
+
 #### `--fix-type`
 
-This option allows you to specify the type of fixes to apply when using either `--fix` or `--fix-dry-run`. The four types of fixes are:
+This option allows you to specify the type of fixes to apply when using either `--fix` or `--fix-dry-run`.
+
+* **Argument Type**: String. One of the following fix types:
+  1. `problem` - fix potential errors in the code
+  1. `suggestion` - apply fixes to the code that improve it
+  1. `layout` - apply fixes that do not change the program structure (AST)
+  1. `directive` - apply fixes to inline directives such as `// eslint-disable`
+* **Multiple Arguments**: Yes
 
-1. `problem` - fix potential errors in the code
-1. `suggestion` - apply fixes to the code that improve it
-1. `layout` - apply fixes that do not change the program structure (AST)
-1. `directive` - apply fixes to inline directives such as `// eslint-disable`
+This option is helpful if you are using another program to format your code but you would still like ESLint to apply other types of fixes.
```

Comment:
```suggestion
This option is helpful if you are using another program to format your code but would still like ESLint to apply other types of fixes.
```

Reply:
```suggestion
This option is helpful if you are using another program to format your code, but you would still like ESLint to apply other types of fixes.
```

i think adding comma is better fix here. clears up parts of the sentence